parm/config/gfs/config.base

```diff
@@ -363,7 +363,7 @@ export binary_diag=".false."
 # Verification options
 export DO_METP="YES"         # Run METPLUS jobs - set METPLUS settings in config.metp
 export DO_FIT2OBS="YES"      # Run fit to observations package
-export DO_VRFY_OCEANDA="NO"  # Run SOCA Ocean DA verification tasks
+export DO_VRFY_OCEANDA="@DO_VRFY_OCEANDA@"  # Run SOCA Ocean and Seaice DA verification tasks
```

Comment:
A default should be added after https://github.com/NOAA-EMC/global-workflow/blob/ba6a4fdf6e245b57530f2b20e6f0ccf567115720/parm/config/gfs/yaml/defaults.yaml#L9 to ensure this gets rendered during experiment setup.

Reply:
:man_facepalming: 
I thought it was already in there ...